src/tools/r2rdump/R2RDump.csproj

```diff
@@ -9,4 +9,8 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="System.CommandLine" Version="0.1.0-e160119-1" />
```

Comment:
Jan noted that this is not the best package to use. I recommend taking care of this after all the other comments.